resources/multisite_drupal_standard.make

```diff
@@ -364,7 +364,7 @@ projects[mailsystem][subdir] = "contrib"
 projects[mailsystem][version] = "2.34"
 
 projects[maxlength][subdir] = "contrib"
-projects[maxlength][version] = "3.2"
+projects[maxlength][version] = "3.2-beta2"
```

Comment:
The version 3.2-beta2 has 2 additional commits that the version 3.2.
This name is confusing.
http://cgit.drupalcode.org/maxlength/log/